### PR TITLE
Fixup

### PR DESCRIPTION
**Pull request by joshmoore**
Combines gh-1 and gh-2 along with a few other modifications.

### CODE REVIEW

**Comment by joshmoore** [comment]
Whoa!

```
 [WARNING]: - omero-user was NOT installed successfully: Role has no field
named u'owner'
```

The output from https://travis-ci.org/openmicroscopy/ansible-public-omero-example/builds/227846660?utm_source=github_status&utm_medium=notification is interesting in its verbosity.

**Comment by manics** [comment]
Can you remove `verbose` and `diff` from https://github.com/joshmoore/ansible-public-omero-example/blob/1d543fb04d14434cb4beda25d71d8a7375103bd5/molecule.yml#L20?

**Comment by joshmoore** [comment]
@manics: have you seen this before?

```
E   TypeError: Can't instantiate abstract class CLI with abstract methods parse, run
```

**Comment by manics** [comment]
@joshmoore Incompatible versions of ansible/molecule?

**Comment by joshmoore** [comment]
I could have _sworn_ I had already moved to `ome-ansible-molecule-dependencies`.

**Comment by joshmoore** [comment]
Well, we're getting somewhere:

```
E           ValueError: No JSON object could be decoded
```

**Comment by kennethgillen** [comment]
I don't want to conflict, but had updated to the official requirements as @manics had asked. Had to make a change to the playbook.yml as ice_version no longer has a default.

cf https://github.com/openmicroscopy/ansible-public-omero-example/compare/master...kennethgillen:working_requirements#diff-2f78c4e27feaf25ffb33d97e8ff7e7a6R27

**Comment by manics** [comment]
@joshmoore Does it pass locally: https://github.com/joshmoore/ansible-public-omero-example/blob/97a498f42cb83df5920400537b98710ffd700cd4/tests/test_default.py#L12

If not `molecule login` and investigate?

**Comment by joshmoore** [comment]
Yup, doing that now. Will push shortly.

**Comment by joshmoore** [comment]
Ha! https://github.com/openmicroscopy/omero-marshal/pull/38

**Comment by joshmoore** [comment]
Testing https://github.com/openmicroscopy/ansible-role-omero-web/pull/4 locally, but we could also potentially tag https://github.com/openmicroscopy/ansible-role-omero-web/pull/3

**Comment by joshmoore** [comment]
@manics PR 4 above passed:

```
# (v2)lifesci-62060:ansible-public-omero-example jamoore$ git diff
diff --git a/requirements.yml b/requirements.yml
index afd1673..c3b8b8d 100644
--- a/requirements.yml
+++ b/requirements.yml
@@ -11,6 +11,8 @@
 - name: openmicroscopy.omero-server

 - name: openmicroscopy.omero-web
+  src:  https://github.com/manics/ansible-role-omero-web.git
+  version: web-dynamic-requirements
```

however I'm pushing with no changes to make use of 1.0.0-m2 (PR 3)

**Comment by joshmoore** [comment]
Finally green!

File: README.md [hunk 1]
```diff
@@ -0,0 +1,15 @@
+OMERO with Public User
+----------------------
+
+This repository recreates the configuration documented under
+https://www.openmicroscopy.org/site/support/omero/sysadmins/public.html
```
Comment (manics) [inline README.md]: Not 100% true, for `omero config set omero.web.public.url_filter` that doc suggests:
- `^/(webadmin/myphoto/|webclient/(?!(action|logout|annotate_(file|tags|comment|rating|map)|script_ui|ome_tiff|figure_script))|webgateway/(?!(archived_files|download_as)))`

whereas this playbook currently has
- `^/(webadmin/myphoto/|webclient/(?!(action|annotate_(file|tags|comment|rating|map)|script_ui|ome_tiff|figure_script))|webgateway/(?!(archived_files|download_as)))`

Want to update the playbook?

File: README.md [hunk 2]
```diff
@@ -0,0 +1,15 @@
...
+
+Playbooks and the associated requirements.yml files can be used for
+creating a basic OMERO configuration with a public user `public`. Most
+likely you will want to copy the configuration into your own playbooks.
+
+### Quick start ###
+
+- Install `ansible` and `ansible-galaxy`: e.g. `pip install ansible ansible-galaxy`
```
Comment (manics) [inline README.md]: Only `ansible`, not `ansible-galaxy`

Comment (kennethgillen) [inline README.md]: You don't need `ansible-galaxy` to get the roles from the requirements file then?

File: README.md (hunk 1 above)
Comment (joshmoore) [inline README.md]: Happy to rationalize. This is an overarching worry. Is there any way (default in etc/omero.properties) that could have this somewhere parseable? cc: @aleksandra-tarkowska @sbesson @will-moore @jburel

Comment (joshmoore) [inline README.md]: Not to mention the needed extensibility so that plugins can add themselves.

File: README.md (hunk 2 above)
Comment (manics) [inline README.md]: It doesn't exist, it's in the ansible package:
```
pip install ansible-galaxy
Collecting ansible-galaxy
  Could not find a version that satisfies the requirement ansible-galaxy (from versions: )
No matching distribution found for ansible-galaxy
```

**Comment by manics** [comment]
> My understanding is we recommend production deployments use versions in requirements.yml.

Others can decide on that

**Comment by joshmoore** [comment]
> My understanding is we recommend production deployments use versions in requirements.yml.

My thinking was that with the current state of things:

 * m1, m2, m3 of omero-server and omero-web
 * brokenness of omero-common on galaxy
 * lack of auto-version-bump functionality in our infrastructure

that it would simplify our lives to hold off just a bit on that front. Eventually I definitely think that travis and/or jenkins should fail if a version is missing (without a reason)

**Comment by joshmoore** [comment]
After discussion with the sysadmin folks, merging and tagging `0.1.0`

**Pull request merged**